sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -91,133 +117,119 @@ Then, activate the environment:
     conda activate bkdev
 
 .. note::
+    To update your local environment, use
+    ``conda env update -f environment.yml``. Updating your local environment
+    is necessary whenever the dependencies in ``environment.yml`` change. This
+    can happen when the environment file is updated in the main Bokeh repository
+    or when you switch branches to work on different issues, for example.
 
-    Before creating a conda environment for the first time, you may need to
-    initialize Anaconda with the ``conda init`` command (only once).
+To learn more about creating and managing conda environments, see `Managing
+environments`_ in the `Conda documentation`_.
 
-.. _dev_guide_installing_node_packages:
+.. _contributor_guide_setup_installing_node_packages:
 
-Installing Node Packages
+4. Install Node packages
 ------------------------
 
 Building BokehJS also requires installing JavaScript dependencies using
-the Node Package Manager. If you have followed the instructions above,
+the `Node Package Manager (npm) <npm_>`_. If you have followed the
+:ref:`instructions above <contributor_guide_setup_creating_conda_env>`,
 ``conda`` has already installed the necessary ``npm`` and ``node.js``
 packages to your system.
 
-Bokeh is typically updated to require the latest major revision of ``npm``
-in order to build. To install the latest version globally, start from the
-top level of the *source checkout* directory, and execute the following
-commands:
+Bokeh usually requires the latest major revision of ``npm``. To install the
+newest version globally, start from the top level of the *source checkout*
+directory, and run the following commands:
 
 .. code-block:: sh
 
     cd bokehjs
     npm install -g npm@7
 
-If you do not wish to install globally (i.e. with ``-g``), then all
-subsequent ``npm`` commands will need to be adjusted to use the local
+If you do not want to install npm globally, leave out the ``-g`` flag. In this
+case, you need to adjust all subsequent ``npm`` commands to use the local
 version installed under ``bokehjs/node_modules``.
 
-Next, still in the ``bokehjs`` subdirectory, execute the following command
-to install all of BokehJS JavaScript dependencies:
+Next, still in the ``bokehjs`` subdirectory, run the following command
+to install all of BokehJS' JavaScript dependencies:
```

Comment:
extreme nit: I try to avoid trailing possessives like this, they read awkwardly to me in text. Maybe it's just me. But FWIW I think any of these would be preferable

> to install all BokehJS JavaScript dependencies:

> to install all the BokehJS JavaScript dependencies:

> to install all the JavaScript dependencies for BokehJS: